src/lifedrain/lifedrain.py

```diff
@@ -822,7 +852,10 @@ def showQuestion():
     if not lifeDrain.disable:
         activateTimer()
         if lifeDrain.status['reviewed']:
-            lifeDrain.deckBarManager.recover()
+            if lifeDrain.status['reviewResponse'] == 1:
+                lifeDrain.deckBarManager.recover(damage=1)
```

Comment:
When you added a new parameter to `recover` method, `damage` was Boolean. Wouldn't be better to keep the same pattern and use `True` instead?

Reply:
Yep, that's what happens when you spend two long writing Perl 😅